templates/named.conf.erb

```diff
@@ -21,6 +21,9 @@ options {
 <%-   end -%>
 	};
 <%- end -%>
+<%- if @forward != '' -%>
```

Comment:
You have a good point.  I think I originally just copied the line from zone.conf.erb, but looking more into it, you've used "<%- if @variable and @variable != '' -%>" as well which may be a better approach.

Also, this being the first time I've worked with a puppet module, will the reuse of @forward (from zone.conf.erb) create a conflict or is it unique to the scope of the conf file?